README.md

```diff
@@ -29,6 +29,45 @@ module.exports = function(config) {
 };
 ```
 
+#### Ignoring stray promises in a test
+
+Stray promises can be ignored in a single test or in a suite by calling `this._ignoreStrayPromises` in the jasmine test context.
+
+```javascript
+function asyncCall() {
+  Promise.resolve('baz')
+  .then(() => { /* do something */ });
+}
+
+describe('foo', function() {
+
+  it('bar', function() {
+    this._ignoreStrayPromises();
+
+    asyncCall();
+  });
+});
+```
+
+#### Other Promise implementations
+
+By default `jasmine-stray-promises` hook into the `window.Promise` implementation, but custom ones can be added as long as they
+implement the thenable interface (i.e. has a "then" method on it's prototype).
```

Comment:
Okay - just mismatching plural and singular, which reads weird to me. Gets the point across though. Maybe at least fix the `it's` => `its`?